soccer-records-model/src/test/java/cz/muni/fi/pa165/team/TeamMatchGoalTest.java

```diff
@@ -0,0 +1,96 @@
+package cz.muni.fi.pa165.team;
+
+import org.testng.annotations.Test;
+
+import java.util.Date;
+
+/**
+ * @author Tomas Smid <smid.thomas@gmail.com>
+ */
+public class TeamMatchGoalTest
+{
+
+    @Test(expectedExceptions = IllegalArgumentException.class)
```

Comment:
když je tady vice mist, ze kterych to může vyletět, chtělo by to ještě `expectedExceptionsMessageRegExp`